lib/msal-common/src/cache/entities/ServerTelemetryEntity.ts

```diff
@@ -5,10 +5,18 @@
 import { SERVER_TELEM_CONSTANTS } from "../../utils/Constants";
 
 export class ServerTelemetryEntity {
-    failedRequests: Array<string|number>;
-    errors: string[];
-    errorCount: number;
-    cacheHits: number;
+    public failedRequests: Array<string|number>;
+    public errors: string[];
+    public cacheHits: number;
+
+    static initializeServerTelemetryEntity(): ServerTelemetryEntity {
```

Comment:
Why not a constructor?

Reply:
Forgot to change this when I updated the design. Thanks for the callout!